history-of-the-quakers-v3/original/29-book-6-chapter-9.adoc

```diff
@@ -200,8 +223,11 @@ London renewed their application to parliament,
 for accepting the solemn affirmation of the members of their society instead of an oath,
 by the following case and petition.
 
-The suffering case of the People commonly called Quakers, relating to oaths and swearing,
-humbly offered.
+[.embedded-content-document.address]
+--
+
+[.blurb]
+=== The suffering case of the People commonly called Quakers, relating to oaths and swearing, humbly offered.
```

Comment:
I seem to not be able to comment where I had left a note for myself to, but lines 294, 297, and 299 look like another letter but they are a case and a petition within the document they are embedded in. I wasn't sure if I should separate them from it or not into their own [embedded-content-document.].

Reply:
Yes, I think you should separate them into another document. 👍